packages/foreman/foreman-fapolicyd/60-foreman.rules

```diff
@@ -0,0 +1,4 @@
+allow perm=open exe=/usr/bin/ruby trust=1 : path=/etc/foreman/encryption_key.rb ftype=text/x-ruby trust=0
```

Comment:
should those files really live in packaging vs a `foreman-fapolicy(d)?` repo?

Reply:
Should is subjective :) I think they are fine here given their simplicity, and that these are potentially specific to RPM ecosystem. However, if others would prefer they live in a stand-alone repository I am happy to go that route.

Reply:
foreman selinux is specific to rpm ecosystem too and lives in an own repo.

I'd prefer repo

Reply:
*whisper* packit *whisper*

(but yeah, it needs to have a spec file first, which I guess is a chicken and egg thing)

Reply:
> _whisper_ packit _whisper_

Touche sir. Tou. che.

Reply:
Moved the rules -- https://github.com/theforeman/foreman-fapolicyd/pull/1